### ISSUE TITLE
v0.16 not available in OpenShift Operator catalog

### ISSUE THREAD

**Issue by caseyrobb** [citrusframework/yaks#481]
v0.16 was released over a week ago and is not yet available in the OpenShift Operator catalog.  Could this be due to the test failure: https://github.com/citrusframework/yaks/actions/runs/6034854151/job/16374060958?

**Comment by christophd** [comment]
yeah, unfortunately 0.15.1 has still not been completely merged yet (https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/3189)
